README.md

```diff
@@ -1,57 +1,129 @@
 ```bash
 sudo apt update && sudo apt install python3-pip
-pip install aiidalab-launch start
 ```
-close the WSL terminal and re-open it
-make sure you have [WSL 2 running](https://docs.docker.com/desktop/wsl/):
 
-oepn a terminal in your mac/linux (or WSL for windows) and execute in a appropriate directory (e.g. your home folder or the home in the WSL terminal)
+**TODO: Add instructions to enable docker within Ubuntu WSL.**
+
+### Install aiidalab-launch
+
+To install aiidalab-launch, please run the following command in your terminal:
+
 ```bash
-git clone https://github.com/nanotech-empa/aiidalab-for-teaching.git
+pip install aiidalab-launch
 ```
-enter the aiidalab-for-teaching folder:
+
+
+### Get the aiidalab-for-teaching image
+
+At this point you can decide to either use a pre-compiled image from the GitHub Container Registry (ghcr.io) or to build the image locally.
+The first latter approach is recommended if you run on a machine with a different architecture than `x86_64` (e.g. ARM).
+
+The following sub-sections require to enter the commands in a terminal (Mac/Linux) or in the Ubuntu WSL terminal (on Windows).
+
+#### Pull the pre-compiled image
+
+To download the pre-compiled image from the GitHub Container Registry, please run the following command:
+
 ```bash
-cd aiidalab-for-teaching
+docker pull ghcr.io/nanotech-empa/aiidalab-for-teaching:main
 ```
 
-Install aiidalab-launch on your PC
+> [!WARNING]
+> The `docker pull` command might not work. To fix that, you need to authenticate to GitHub first:
+> ```bash
+> docker login ghcr.io -u <github-username>
+> Password: <your-token>
+> ```
+> The access token can be created [here](https://github.com/settings/tokens).
+> While creating it, please make sure to enable the following permissions:
+> ```
+> * write:packages
+> * read:packages
+> * delete:packages
+> ```
+
+To rename the image to `aiidalab/teaching`, please run the following command:
 
-execute:
 ```bash
-pip install aiidalab-launch
+docker tag ghcr.io/nanotech-empa/aiidalab-for-teaching:main aiidalab/teaching
 ```
-### Make sure Docker desktop is running
 
-To build the container locally run the following command:
+
+#### Build the image locally
+
+First, make sure the Docker Desktop is running.
+Enter a folder where you want to keep the aiidalab-for-teaching repository (`cd /path/to/your/folder`) and run the following command:
+
 ```bash
-docker build -t aiidalab/teaching .
+git clone https://github.com/nanotech-empa/aiidalab-for-teaching.git
 ```
+Then enter the repository folder (`cd aiidalab-for-teaching`) and build the container by running the following command:
+execute:
 
-### To use a pre-compiled docker image (e.g. if docker build fails):
-run the command:
 ```bash
- docker pull ghcr.io/nanotech-empa/aiidalab-for-teaching:main
+docker build -t aiidalab/teaching .
 ```
-and rename the image to aiidalab/teaching use:
+
+### Install aiidalab-launch
```

Comment:
This was in part already present at line 60

Reply:
thanks, fixed in https://github.com/nanotech-empa/aiidalab-for-teaching/pull/18/commits/ae084cc33aad230e8edf4bbf4d21d709f4ff5007